### ISSUE TITLE
Can't flash after update to 1.5.1

### ISSUE THREAD

**Issue by otacon64** [BotoX/xiaomi-m365-firmware-patcher#42]
Unfortunately I've updated my scooter to firmware v1.5.1 and now I can't flash custom firmware. Is there any plans for support of v1.5.1?

**Comment by raypettersen** [comment]
I got mine shipped with 1.5.1
Is anyone working on the problem?

**Comment by BotoX** [comment]
Need to flash the BLE module with ST-Link -> https://t.me/XiaomiM365Hacking/28567

**Comment by Nojevah** [comment]
The other solution is to wait. There are good chances new signatures/keys used by BLE 0.8.1 will be decrypted. If BotoX is still in business at that time, you can expect a new version. :stuck_out_tongue_winking_eye:

**Comment by snackman1** [comment]
Just found the solution on how to downgrade the BLE back to 072.  Install the Beta version of M365DownG and it will allow you to flash the BLE072.bin.

https://rollerplausch.fandom.com/wiki/Flashing_Firmware

**Comment by Nojevah** [comment]
And if you're patient enough (recommended), you wait for official release on Play Store.

**Comment by snackman1** [comment]
The current version of M365downG in the Google Play store should be able to communicate with the bluetooth driver 081.  Find and download the BLE072.BIN (bluetooth 072 driver) then flash it and you should be downgraded from 081 back to 072.   That's it.

**Comment by guyromb** [comment]
You are awesome guys! 
Works perfectly fine now :-)

**Comment by fabiohbarbosa** [comment]
Guys my scooter is the model 16133 European Black (2 Fuses).

Can I apply the BLE downgrade following the link?https://rollerplausch.fandom.com/wiki/Flashing_Firmware

Which model is your @guyromb ?

Thanks :D

**Comment by MarlonMrN** [comment]
Yes you can @fabioh, almost nobody has the Chinese old 1 fusion version. And those who have it, cannot flash cfw.
You are fine, go for it.

**Comment by zalgocykel** [comment]
> 
> 
> Guys my scooter is the model 16133 European Black (2 Fuses).
> 
> Can I apply the BLE downgrade following the link?https://rollerplausch.fandom.com/wiki/Flashing_Firmware
> 
> Which model is your @guyromb ?
> 
> Thanks :D

What happened to the link? It's broken :/

Anyone saved the content that was on that page?

**Comment by fedya** [comment]
I need help here.
Segway app updated my bluetooth mode to something => 081
and now i need to push scooter button to pair devices.
Any idea how i can disable it? Want to downgrade back.

**Comment by zalgocykel** [comment]
> 
> 
> I need help here.
> Segway app updated my bluetooth mode to something => 081
> and now i need to push scooter button to pair devices.
> Any idea how i can disable it? Want to downgrade back.

I finally found a working link here: https://rollerplausch.fandom.com/de/wiki/Firmware_flashen
Here is the bluetooth bin: http://rollerplausch.com/dl/BLE072.bin
Generate all kinds of FW (including stock): https://m365.botox.bz/

I had my firmware at 1.5.1, what I did (with M365downG):
1. Downgrade bluetooth version
2. Then I went from 1.5.1 ->1.4.3  (stock FW) -> 1.4.0 (stock FW) -> 1.3.8 CFW

**Comment by wjurban002** [comment]
update i figured out a program that flashed to 1.1.7 it was nine tool but now still no app will connect and flash my ninebot s
